docs/guides/mozilla-sops.md

```diff
@@ -19,24 +19,30 @@ brew install gnupg sops
 
 ## Generate a GPG key
 
-Generate a GPG key with OpenPGP without specifying a passphrase:
+Generate a GPG/OpenPGP key with no passphrase (`%no-protection`):
 
 ```console
-$ gpg --full-generate-key
-
-Real name: stefanprodan
-Email address: stefanprodan@users.noreply.github.com
-Comment:
-You selected this USER-ID:
-    "stefanprodan <stefanprodan@users.noreply.github.com>"
+gpg --batch --full-generate-key <<EOF
+%no-protection
+Key-Type: 1
+Key-Length: 4096
+Subkey-Type: 1
+Subkey-Length: 4096
+Expire-Date: 0
+Name-Comment: flux secrets
+Name-Real: cluster0.yourdomain.com
```

Comment:
```suggestion
Name-Real: <cluster0.yourdomain.com>
```

to visualize fields that likely require modifications (may want to do this for `Name-Comment` a well to e.g. add the cluster name to the key name or something).